packages/aws-cdk-lib/aws-ecs/lib/base/base-service.ts

```diff
@@ -881,12 +907,17 @@ export abstract class BaseService extends Resource
         port: svc.port || containerPort,
         dnsName: svc.dnsName,
       };
+      const timeout = (svc.idleTimeout || svc.perRequestTimeout) ? {
```

Comment:
We should check that if any of these values are specified, they should be at least 1 second to prevent unhandled exceptions.
We could create a separate method to keep this more organized:
```ts
private renderTimeout(idleTimeout?: Duration, perRequestTimeout?: Duration) {
  if (!idleTimeout && !perRequestTimeout) return undefined
  if (idleTimeout && idleTimeout.toMilliseconds() > 0 && idleTimeout.toMilliseconds() < Duration.seconds(1)) {
    throw new Error(`idleTimeout must be at least 1 second or 0 to disable it, got ${idleTimeout.toMilliseconds()}ms.`)
  }
  // same for perRequestTimeout...
  return {
    idleTimeoutSeconds: svc.idleTimeout?.toSeconds(),
    perRequestTimeoutSeconds: svc.perRequestTimeout?.toSeconds(),
  }
}
```
?